charts/guac/values.yaml

```diff
@@ -94,7 +94,7 @@ guac:
       # if not set appVersion field from Chart.yaml is used
       tag: ""
       # When digest is set to a non-empty value, images will be pulled by digest (regardless of tag value).
-      digest: "sha256:770ad1e9b6300e27ea3ac03d3808f4fbfdec5e78e0f82019d18d6fd3b3943d22"
```

Comment:
is there a way to keep this up to date with the image released on the GUAC repo? Seems like this will continue to happen.

Reply:
Once guac have a release cycle established we can track that and update accordingly or instruct people to override the image digest in their config. Or we can default to use the latest tag if guac creates it.

We really need the first release of guac to be published as an anchor for the chart to be tested with a known/static version of the code.